eskip/parser.y

```diff
@@ -69,17 +67,42 @@ func convertNumber(s string) float64 {
 %token openarrow
 %token closearrow
 
+%token start_document;
+%token start_predicates;
+%token start_filters;
+
 %%
 
+start:
+	start_document document {
+		eskiplex.(*eskipLex).routes = $2.routes
+	}
+	|
+	start_predicates {
+		// allow empty or comments only
+		eskiplex.(*eskipLex).predicates = nil
+	}
```

Comment:
We can disallow comments for ParsePredicates and ParseFilters by removing these empty rules, see also https://github.com/zalando/skipper/pull/2866